main_routers/memory_router.py

```
@@ -135,6 +135,8 @@ def validate_catgirl_name(name: str, allow_dots: bool = False) -> tuple[bool, st
         return False, "名称不能仅由点号组成或以点号结尾"
     if result.code == "reserved_device_name":
         return False, "名称不能使用 Windows 保留设备名"
+    if result.code == "reserved_route_name":
+        return False, "此名称是系统保留的路由名称，不能用作名称"
```

Ruff (0.15.9)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)